bpf/process/types/basic.h

```diff
@@ -2211,6 +2215,12 @@ do_action(void *ctx, __u32 i, struct selector_action *actions,
 			e->common.flags |= MSG_COMMON_FLAG_USER_STACKTRACE;
 			e->user_stack_id = get_stackid(ctx, &stack_trace_map, BPF_F_USER_STACK);
 		}
+#ifdef __LARGE_MAP_KEYS
+		__u32 ima_hash = actions->act[++i];
+
+		if (ima_hash)
+			e->common.flags |= MSG_COMMON_FLAG_IMA_HASH;
```

Comment:
any reason why not to do separate action? IIUC it's just post action flag now, right?  I don't have strong opinion about that, just curious

Reply:
From my point of view, hash collection is like stacktrace collection. We already have a flag for it. With `NoPost` event neither hash collection nor stacktrace collection makes sense, because we need to post collected information in event. So, that's why I think Post action flag suits better than separate action.